scripts/kafka_bridge_config_generator.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+SECURITY_PROTOCOL=PLAINTEXT
+
+if [ "$KAFKA_BRIDGE_TLS" = "true" ]; then
+    SECURITY_PROTOCOL="SSL"
+
+    if [ -n "$KAFKA_BRIDGE_TRUSTED_CERTS" ]; then
+        TLS_CONFIGURATION=$(cat <<EOF
+# TLS / SSL
+ssl.truststore.location=/tmp/kafka/bridge.truststore.p12
+ssl.truststore.password=${CERTS_STORE_PASSWORD}
+ssl.truststore.type=PKCS12
+EOF
+)
+    fi
+
+    if [ -n "$KAFKA_BRIDGE_TLS_AUTH_CERT" ]; then
+        TLS_AUTH_CONFIGURATION=$(cat <<EOF
+ssl.keystore.location=/tmp/kafka/bridge.keystore.p12
+ssl.keystore.password=${CERTS_STORE_PASSWORD}
+ssl.keystore.type=PKCS12
+EOF
+)
+    fi
+fi
+
+if [ -n "$KAFKA_BRIDGE_SASL_USERNAME" ] && [ -n "$KAFKA_BRIDGE_SASL_PASSWORD_FILE" ]; then
+    if [ "$SECURITY_PROTOCOL" = "SSL" ]; then
+        SECURITY_PROTOCOL="SASL_SSL"
+    else
+        SECURITY_PROTOCOL="SASL_PLAINTEXT"
+    fi
+
+    PASSWORD=$(cat /opt/kafka/bridge-password/$KAFKA_BRIDGE_SASL_PASSWORD_FILE)
+
+    if [ "x$KAFKA_BRIDGE_SASL_MECHANISM" = "xplain" ]; then
+        SASL_MECHANISM="PLAIN"
+        JAAS_SECURITY_MODULE="plain.PlainLoginModule"
+    elif [ "x$KAFKA_BRIDGE_SASL_MECHANISM" = "xscram-sha-512" ]; then
+        SASL_MECHANISM="SCRAM-SHA-512"
+        JAAS_SECURITY_MODULE="scram.ScramLoginModule"
+    fi
+
+    SASL_AUTH_CONFIGURATION=$(cat <<EOF
+sasl.mechanism=${SASL_MECHANISM}
+sasl.jaas.config=org.apache.kafka.common.security.${JAAS_SECURITY_MODULE} required username="${KAFKA_BRIDGE_SASL_USERNAME}" password="${PASSWORD}";
+EOF
+)
+fi
```

Comment:
all the above configuration are needed for both producer and consumer and they have to have `kafka.consumer.` and `kafka.producer.` prefix.
Comparing with Kafka Mirror Maker configuration, there we have two different config generator scripts.